tests/src/Kernel/Schema/AlterableComposableTestSchema.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\Tests\graphql\Kernel\Schema;
+
+use Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException;
+use Drupal\graphql\GraphQL\ResolverRegistry;
+use Drupal\graphql\Plugin\GraphQL\Schema\AlterableComposableSchema;
+
+/**
+ * GraphQL alterable test schema.
+ *
+ * @Schema(
+ *   id = "alterable_composable_test",
+ *   name = "Alterable composable test schema"
+ * )
+ */
+class AlterableComposableTestSchema extends AlterableComposableSchema {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getResolverRegistry() {
+    return new ResolverRegistry();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getExtensions() {
+    return $this->extensionManager->getExtensions($this->getPluginId());
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getSchemaDefinition() {
+    $id = $this->getPluginId();
+    $definition = $this->getPluginDefinition();
+    $module = $this->moduleHandler->getModule($definition['provider']);
+    $path = 'graphql/' . $id . '.graphqls';
+    $file = $module->getPath() . '/' . $path;
+
+    if (!file_exists($file)) {
```

Comment:
why do we need this code? We never read files in the test?

Reply:
Thanks, will remove it.